examples/12_css/vite.config.ts

```diff
@@ -0,0 +1,9 @@
+import url from 'node:url';
+import path from 'node:path';
+import { defineConfig } from 'vite';
+import { vanillaExtractPlugin } from '@vanilla-extract/vite-plugin';
+
+export default defineConfig({
+  root: path.dirname(url.fileURLToPath(import.meta.url)),
```

Comment:
I think we can omit this. (Can be a separate PR, as I might be wrong, and there are other places like this.)